README.md

```diff
@@ -133,21 +133,21 @@ Supported Platforms
 
 ### herokuish_version
 
-- default: `0.5.18`
+- default: `unpinned`
 - type: `version`
-- description: The version of herokuish to install
+- description: The version of herokuish to install.
 
 ### plugn_version
 
-- default: `0.5.0`
+- default: `unpinned`
 - type: `version`
-- description: The version of plugn to install
+- description: The version of plugn to install.
 
 ### sshcommand_version
 
-- default: `0.11.0`
+- default: `unpinned`
```

Comment:
Might be worth modifying readme generation to include valid values or example values?

Reply:
Also maybe instead of a default of a string, just empty string? How do other libraries handle this?

Reply:
> Also maybe instead of a default of a string, just empty string? How do other libraries handle this?

I was also unsure about the default value. I started with `null`, then went to `latest` (which is not a good fit, since in ansible it implies always *updating* to the latest version) and then went for `unpinned`.

Looking around, I notice that most of geerlingguy's roles ([nginx](https://github.com/geerlingguy/ansible-role-nginx/blob/master/defaults/main.yml), [postgresql](https://github.com/geerlingguy/ansible-role-postgresql/blob/master/defaults/main.yml), [apache](https://github.com/geerlingguy/ansible-role-apache/blob/master/defaults/main.yml),  don't allow any version specifier - it will simply install what apt/yum/... installs.
Some are intermediate, e.g. the [docker](https://github.com/geerlingguy/ansible-role-docker/blob/master/defaults/main.yml) role allows to specify a version for `docker-compose` but not for `docker` itself.

Removing the version variables entirely would not only prevent people from choosing/pinning versions via ansible but also from performing an update of dokku via ansible - it would need to be done manually, which is not ideal.

An intermediate solution would be to switch from `<package>_version` to a `<package>_state` variable that only accepts `present` or `latest` (like in the [geerlingguy.apache role](https://github.com/geerlingguy/ansible-role-apache/blob/0c45cec060ea9bd6668634cc18ed46cb51ced807/defaults/main.yml#L58)).
This would allow people to update the system via ansible.

How about this:
 * We keep the `<package>_version` variables for backwards compatibility for the moment. We change their default value to `null`, and if someone sets a different value, we print a deprecation warning
 * We introduce a `dokku_packages_state` variable with default value `present` (also accepts `latest`).

Reply:
sounds good